app/controllers/job_messages_controller.rb

```diff
@@ -13,7 +13,7 @@ def get
     elsif status.read.empty?
       flash_message(:error, t('poll_job.failed'))
       session[:job_id] = nil
-      render 'shared/http_status', locals: { code: '404', message: t('poll_job.not_enqueued') }, status: 404
+      render json: { code: '404', message: t('poll_job.not_enqueued') }, status: 404
```

Comment:
Just to confirm, did you change this so that it's the same type as the normal case (Line 20)?

Reply:
Yes. Also for some (unknown?) reason, the previous version caused any rspec test that ran that line to hang indefinitely.